gui/src/templates/main.html

```diff
@@ -0,0 +1,30 @@
+<!DOCTYPE html>
+<html>
+    <head>
+
+        <title>DOORS | DiReCtEd OpEn ObJeCt ReAliTY SiMuLaToR</title>
```

Comment:
```suggestion
        <title>DOORS | Directed Object Reality Simulator</title>
```
This is a serious matter!

Reply:
Lol, fair enough. Do we want to do `Object` (`DORS`) or `Open Object` (`DOORS`)?

Reply:
Well open object is nod to the fact that it's an opening door that shows what's behind it.